db/migrate/20161025021845_remove_staff_number_from_people.rb

```diff
@@ -0,0 +1,21 @@
+class RemoveStaffNumberFromPeople < ActiveRecord::Migration
+  def change
+    remove_column :people, :staff_number, :string
+
+    reversible do |dir|
+      Person.reset_column_information
+
+      dir.down do
+        Person.transaction do
+          # Grab the staff number from the family
+          Person.all.each do |person|
+            if (staff_number = person.family.try(:staff_number))
+              person.staff_number = staff_number
```

Comment:
When I try to migrate down I get an error: 

```
== 20161025021845 RemoveStaffNumberFromPeople: reverting ======================
rake aborted!
StandardError: An error has occurred, this and all later migrations canceled:

undefined method `staff_number=' for #<Attendee:0x007fc725910b10>
```

I believe this is due to the order of operations here, the `remove_column` should be placed below `reversible` so that when migrating down the first step (in reverse) will be to add the column.